client/my-sites/media-library/list.jsx

```diff
@@ -158,16 +158,15 @@ export class MediaLibraryList extends React.Component {
 		const itemDate = new Date( date );
 		const currentDate = new Date();
 
-		if ( itemDate.getYear() === currentDate.getYear() ) {
-			return moment( date ).format( 'MMM DD' );
+		if ( itemDate.getFullYear() === currentDate.getFullYear() ) {
+			return moment( date ).format( 'MMM D' );
 		}
 
-		return moment( date ).format( 'MMM DD, YYYY' );
+		return moment( date ).format( 'MMM D, YYYY' );
```

Comment:
This, updates this to day without a leading zero right?

Reply:
Indeed 🙂.